doc/api/fs.md

```diff
@@ -1069,6 +1069,38 @@ including subdirectories and files.
 When copying a directory to another directory, globs are not supported and
 behavior is similar to `cp dir1/ dir2/`.
 
+### `fsPromises.glob(pattern[, options])`
```

Comment:
This is experimental so this is fine - but in general when we add a promise API we probably want to also add a callback API